### DISCUSSION TITLE
Work with education app

### DISCUSSION THREAD

**Discussion by amerwals** [lavaloon-eg/ksa_compliance#182] [Q&A]
when try to create sales invoices by education app i should add Tax Exemption Reason and national id in it, how can i modify it by add line "BT-46-1 = NAT" to void get reject.
i got that reject code:
{"validationResults":{"infoMessages":[{"type":"INFO","code":"XSD_ZATCA_VALID","category":"XSD validation","message":"Complied with UBL 2.1 standards in line with ZATCA specifications","status":"PASS"}],"warningMessages":[],"errorMessages":[{"type":"ERROR","code":"BR-KSA-49","category":"KSA","message":"[BR-KSA-49]- **If the tax exemption reason code (BT-121) is equal to VATEX-SA-EDU or VATEX-SA-HEA, then the other buyer ID (BT-46) is mandatory and must be national ID (BT-46-1 = NAT).","status":"ERROR"}**

**Comment by mhaggag** [comment]
Closing the discussion since github issue #183 is tracking this.
